jwt-mac/src/test/scala/tsec/JWTMacTests.scala

```diff
@@ -59,6 +60,24 @@ class JWTMacTests extends TestSpec with MustMatchers {
       res.unsafeRunSync() mustBe true
     }
 
+    it should "parse unverified correctly" in {
+      val res = for {
+        key    <- keyGen.generateKey
+        claims <- JWTClaims.withDuration[IO](notBefore = Some(-2.seconds), expiration = Some(10.seconds))
```

Comment:
Hm, I see negative seconds the first time, is this useful ?

Reply:
`notBefore` will just give you some `now.plusSeconds(t)` when the token can't be used.

This simply ensures after callign that function, `notBefore` always is valid.